plugins/database/src/general_query_setup.cpp

```diff
@@ -225,6 +226,7 @@ icatGeneralQuerySetup() {
     sColumn( COL_R_RESC_STATUS, "R_RESC_MAIN", "resc_status" );
     sColumn( COL_R_CREATE_TIME, "R_RESC_MAIN", "create_ts" );
     sColumn( COL_R_MODIFY_TIME, "R_RESC_MAIN", "modify_ts " );
+    sColumn( COL_R_MODIFY_TIME_MILLIS, "R_RESC_MAIN", "modify_ts_millis" );
```

Comment:
does this appear in `iadmin attrs` automatically as well?

Reply:
The GenQuery name does.

Reply:
perfect - thanks.